docs/commands/pipeline/create.md

```diff
@@ -0,0 +1,53 @@
+# pipeline create
+
+Create an update for dataset dependency in Studio
+
+## Synopsis
+
+```usage
+usage: datachain pipeline create [-h] [-v] [-q]
+                               [-V VERSION]
+                               [-r] [-n NAMESPACE]
+                               [-p PROJECT]
+                               [-t TEAM]
+                               dataset
+```
+
+## Description
+
+This command creates an pipeline for a dataset dependency. The pipeline is determined based on the dependency for the dataset in Studio and is created accordingly. The dataset name can be a fully qualified name including the namespace and project, or a regular name, in which case the explicitly defined namespace and project will be used if they are set; otherwise, default values will be applied.
+
+## Arguments
+
+* `dataset` - Name of the dataset (can be a fully qualified name including the namespace and project or regular name)
+
+## Options
+
+* `-V VERSION, --version VERSION` - Version of the dataset (default: latest)
+* `-r, --review` - Review the pipeline before creating. (Opens in paused state)
```

Comment:
as I mentioned in Slack - let's get rid of this complexity. Create always in "review" mode.